GalaxyZooTouchTable/GalaxyZooTouchTable/Models/TableUserFactory.cs

```diff
@@ -36,51 +36,51 @@ public bool Active
         }
     }
 
-    public class PersonUser : TableUser
+    public class PurpleUser : TableUser
     {
-        public override string Name => "PersonUser";
+        public override string Name => "PurpleUser";
         public override string ThemeColor => "#A5A2FB";
-        public override BitmapImage StartButton => new BitmapImage(new Uri("../Images/StartIcon/Person.png", UriKind.Relative));
-        public override BitmapImage Avatar => new BitmapImage(new Uri("../Images/Avatars/Person.png", UriKind.Relative));
+        public override BitmapImage Avatar => new BitmapImage(new Uri("../Images/Avatars/purple.png", UriKind.Relative));
+        public override string DashArray => "0.25 1.5 2 1.5";
```

Comment:
As per the comment above, in here is where I might put those `XPosition` and `YPosition` attributes.